### PR TITLE
Add deployment logs to UI

### PR DESCRIPTION
**Pull request by spadgett**
Add a Logs tab to the deployment page.

screen shot 2015-11-11 at 2 43 45 pm

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres PTAL

**Comment by jwforres** [comment]
only concern is RCs that arent deployments are now going to have a snaggle-tooth Details tab :-\

**Comment by spadgett** [comment]
Not sure how I feel about it, but you could set `display: none;` on `.nav-tabs` to hide just the tabs when there's only one.

**Comment by jwforres** [comment]
Can we just hit the tab or do we have to hide the whole nav-tabs because of
the horizontal line?   if its just the tab we could use the :only-child
selector

**Comment by spadgett** [comment]
@jwforres I have a change that gets rid of the snaggletooth tab without CSS hacks... PTAL

File: assets/app/scripts/app.js [hunk 1]
```diff
@@ -139,11 +139,21 @@ angular
...
+
+          return 'views/browse/deployment.html';
+        },
+        controller: 'DeploymentController'
       })
       // Needs to still be prefixed with browse/deployments so the secondary nav active state is correct
       .when('/project/:project/browse/deployments-replicationcontrollers/:replicationcontroller', {
-        templateUrl: 'views/browse/deployment.html'
+        templateUrl: function(params) {
+          return 'views/browse/replication-controller.html';
```
Comment (jwforres) [inline assets/app/scripts/app.js]: when templateUrl doesnt need any logic, dont set it to a function just set it to the value

**Comment by jwforres** [comment]
just the one nit, otherwise LGTM

**Comment by spadgett** [comment]
@jwforres thanks, updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**